pkg/apis/kubermatic/v1/datacenter.go

```diff
@@ -98,19 +97,39 @@ type SeedSpec struct {
 	// Metering configures the metering tool on user clusters across the seed.
 	Metering *MeteringConfiguration `json:"metering,omitempty"`
 	// BackupRestore when set, enables backup and restore controllers with given configuration.
+	// Deprecated: use EtcdBackupRestore instead which allows for multiple destinations. For now, it's still supported and
+	// will work if set.
 	BackupRestore *SeedBackupRestoreConfiguration `json:"backupRestore,omitempty"`
+	// EtcdBackupRestore holds the configuration of the automatic etcd backup restores for the Seed
+	EtcdBackupRestore *EtcdBackupRestore `json:"etcdBackupRestore,omitempty"`
 }
 
-// SeedBackupRestoreConfiguration are s3 settings used for backups and restores of user cluster etcds.
+// SeedBackupRestoreConfiguration defines the bucket name and endpoint as a backup destination.
+// Deprecated: use EtcdBackupRestore
 type SeedBackupRestoreConfiguration struct {
-	// +kubebuilder:default=s3.amazonaws.com
```

Comment:
is it not defaults anymore?

Reply:
It never defaulted it anywhere, it was misleading unfortunately.